rclrs/src/context.rs

```diff
@@ -118,6 +127,47 @@ impl Context {
     }
 }
 
+/// Returns non-ROS arguments held by context.
+///
+/// This function should be called after rcl_init. `rcl_arguments` should be `global_arguments`
+/// field of initialized context. `args` is array of input arguments passed to node.
+fn get_non_ros_arguments(
+    rcl_arguments: *const rcl_arguments_t,
+    args: &[*const c_char],
+) -> Result<Vec<String>, RclrsError> {
+    // SAFETY: In case of error -1 will be returned and handled a line below.
+    let args_status = unsafe { rcl_arguments_get_count_unparsed(rcl_arguments) };
+    if args_status == -1 {
+        return Err(RclrsError::RclError {
+            code: InvalidArgument,
+            msg: None,
+        });
+    }
+    // SAFETY: All possible negative args_status values handled above.
+    let args_num = args_status as usize;
+    if args_num == 0 {
+        return Ok(Vec::new());
+    }
```

Comment:
The `as` keyword signifies a safe cast, so this cannot cause undefined behavior. As such, a safety comment isn't needed here (those should be for dealing with `unsafe` and UB.

HOWEVER, it's good to document why the cast isn't going to cause a logic error.
```suggestion
    // All possible negative args_status values were handled above.
    let args_num = args_status as usize;
    if args_num == 0 {
        return Ok(Vec::new());
    }
```